docs/src/features.md

```diff
@@ -176,17 +230,57 @@ For the connection from the hub to the demand there are associated transmission
+& \qquad flow_{(balance,demand),4:6} \leq p^{init\_export\_capacity}_{(balance,demand)} \\
 \end{aligned}
 ```
 
 ```math
 \begin{aligned}
 & min\_transport\_flows\_limit_{(balance,demand),1:3}: \\
-& \qquad flow_{(balance,demand),1:3} \geq - p^{import\_capacity}_{(balance,demand)} \\
+& \qquad flow_{(balance,demand),1:3} \geq - p^{init\_import\_capacity}_{(balance,demand)} \\
 & min\_transport\_flows\_limit_{(balance,demand),4:6}: \\
-& \qquad flow_{(balance,demand),4:6} \geq - p^{import\_capacity}_{(balance,demand)} \\
+& \qquad flow_{(balance,demand),4:6} \geq - p^{init\_import\_capacity}_{(balance,demand)} \\
+\end{aligned}
+```
+
+### Storage Level limits
+
+Since we have a storage asset in the system, we need to limit the maximum storage level. The `phs` time resolution is defined for each six hours, so we only have one constraint.
+
+```math
+\begin{aligned}
+& max\_storage\_level\_limit_{phs,1:6}: \\
+& \qquad storage\_level_{phs,1:6} = p^{init\_storage\_capacity}_{phs}
 \end{aligned}
 ```
+
+## [Comparison of Different Modeling Approaches](@id comparison)
+
+This section quantifies the advantages of the [`flexible connection`](@ref flex-asset-connection) and [`flexible time resolution`](@ref flex-time-res) in the *TulipaEnergyModel.jl* modeling approach. So, let us consider three different approaches based on the same example:
+
+1. Classic approach with hourly resolution: This approach need and extra asset, called `node`, to create the hybrid operation of the `phs` and `wind` assets.
+2. Flexible connection with hourly resolution: This approach use the flexible connection to represent the hybrid operation of the  `phs` and `wind` assets.
+3. Flexible connection and time resolution: This approach use both features, the felxible connection and the flexible time resolution.
+
+> **Note:** *TulipaEnergyModel.jl* is flexible enough to allow any of these three approaches to the model through the input data.
+
+The table below shows the number of constraints and variables for each approach over a six-hour horizon. This highlights the potential of flexible time resolution in reducing the size of the optimization model.
+
+| Modeling approach                          | Nº Variables | Nº Constraints | Objective Function |
+|:-------------------------------------------|:------------ |:---------------|:------------------ |
+| Classic approach with hourly resolution    | 48           | 84             | 28.4365            |
+| Flexible connection with hourly resolution | 42           | 72             | 28.4365            |
+| Flexible connection and time resolution    | 16           | 25             | 28.4587            |
+
+By comparing the classic approach with the other methods, we can analyze their differences:
+
+- The flexible connection with hourly resolution reduces 6 variables ($12.5\%$) and 12 constraints ($\approx 14\%$). The objective function is the same since in both cases we use an hourly time resolution.
```

Comment:
The percentage sign does not seem to show correctly in my browser, maybe you can see the correct behavior in your local version.

Reply:
It is the same as in the `#hide` thing; you need to create it locally to see how it renders. At least they render correctly for me locally, so I assume they will do the same in GitHub when the documentation is created.